services/collaboration/pkg/helpers/path.go

```diff
@@ -0,0 +1,41 @@
+package helpers
+
+import (
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/owncloud/ocis/v2/services/collaboration/pkg/config"
+)
+
+// ParseWopiFileID extracts the file id from a wopi path
+//
+// If the file id is a jwt, it will be decoded and the file id will be extracted from the jwt claims.
+// If the file id is not a jwt, it will be returned as is.
+func ParseWopiFileID(cfg *config.Config, path string) string {
```

Comment:
maybe it's better to move it to the middleware/wopicontext.go file, as a private method. Not sure if the method will be use in any other place

Reply:
fine.